tests/unit/test_webhook_views.py

```diff
@@ -678,6 +601,8 @@ def test_webhooks_reload_master(api_app, users, access_token, json_headers,
                                 online_video, api_project, datadir,
                                 mock_sorenson):
     """Test webhooks reload master after publish/edit/publish."""
+    api_app.config['DEPOSIT_DATACITE_MINTING_ENABLED'] = False
```

Comment:
test not passing without this, it is calling DataCite otherwise... if you have any idea...